app/models/booking.rb

```diff
@@ -1,5 +1,5 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :workshop
-  belongs_to :review
+  has_many :reviews
```

Comment:
unsure about that, one booking only has one review, right? so workshops can have multiple reviews but only one for each booking i would think.

Reply:
Yes you're right, I was mixing up bookings and workshops. It should be a one to one relationship. So I think review will belongs_to :booking and then the booking model will have no mention of review, right?